deploy/crds/metal3_v1alpha1_baremetalhost_crd.yaml

```diff
@@ -0,0 +1,387 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
How is this file different from deploy/crds/metal3.io_baremetalhosts_crd.yaml

Do we need both? Was this generated automatically?

Reply:
I think the file was renamed at some point. This looks like a rebase artifact.

Reply:
To me it seems like the this PR has turned out to be obsolete. After several rebases the affected code is quite different compared to where it started from and thus the whole task should be reconsidered.
I would propose to close this PR.
What do you think?